webapp/src/immersivereader.tsx

```diff
@@ -60,11 +61,11 @@ function beautifyText(content: string): string {
         );
     }
 
-    // If the ```block ``` annotation is used to display a block in
-    // a modal, remove it
-    function cleanHintText(content: string): string {
+    // Remove any triple tick annotations ```
+    // This RegEx was copied from pxtlib/tutorial.ts
```

Comment:
Should probably drop a note there or we'll never remember to update this -- might be easy to put a function in `pxtlib/tutorial.ts` that returns this regex so we don't have to dup it though